pkg/supplemented/supplemented.go

```diff
@@ -0,0 +1,392 @@
+package supplemented
```

Comment:
supplemented?
Not exactly a common term...

Reply:
I'm open to alternatives, so long as they're at least kind of descriptive.